acquia/acquia_cms
# ACMS 1099: Make content type reference fields optional

**Motivation**
Fixes #[ACMS-1099](https://backlog.acquia.com/browse/ACMS-1099)

**Proposed changes**
Make content type reference fields optional.
For article content type display author field should be optional.

**Alternatives considered**
N/A

**Testing steps**
- Login to https://orionacmsdev.prod.acquia-sites.com as administrator
- Head towards /node/add/article
- Verify that Display Author field is now optional
- Verify you are able to create article without author

**Merge requirements**
